client/src/components/ui/filter-dropdown.tsx

```diff
@@ -0,0 +1,86 @@
+import { useState } from 'react'
+import {Button, Popover, Selection, MenuTrigger, Menu, MenuItem} from 'react-aria-components';
+import { ExpandMore, ExpandLess, CheckBox, CheckBoxOutlineBlank } from '@mui/icons-material';
+import { MenuItemProps } from '@mui/material/MenuItem';
```

Comment:
❔ I'm a bit confused here. I would expect that we need to import from react-aria, but it seems to be working 🤔

Can you provide any insight on this one?